MekHQ/src/mekhq/campaign/mission/ScenarioMapParameters.java

```diff
@@ -11,14 +11,28 @@
  * @author NickAragua
  *
  */
-public class ScenarioMapParameters {
+public class ScenarioMapParameters implements Cloneable {
     public enum MapLocation {
         AllGroundTerrain,
         SpecificGroundTerrain,
         Space,
         LowAtmosphere
     }
     
+    public Object clone() {
```

Comment:
I _think_ you should be able to make this return `ScenarioMapParameters`, as that is a more derived type.
```suggestion
    @Override
    public ScenarioMapParameters clone() {
```